src/app/CASESessionManager.h

```diff
@@ -89,7 +89,7 @@ class CASESessionManager : public Dnssd::OperationalResolveDelegate
      * The results of the DNS-SD resolution request is provided to the class via `OperationalResolveDelegate`
      * implementation of CASESessionManager.
      */
-    CHIP_ERROR ResolveDeviceAddress(FabricInfo * fabric, NodeId nodeId);
+    CHIP_ERROR ResolveDeviceAddress(FabricInfo * fabric, NodeId nodeId, bool establishSession = false);
```

Comment:
But this doesn't actually establish a session, in the sense of doing CASE, right?  All it does is create an OperationalDeviceProxy....